src/backend/web/routes/login.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express');
+const session = require('express-session');
+const bodyParser = require('body-parser');
+const cookieParser = require('cookie-parser');
+const passport = require('passport');
```

Comment:
My guess is that later we're going to want to move some of this further up the router stack, so other parts of the web app can use it.  For now, this is OK.

Reply:
I was thinking the same, I just don't want it affecting any of the additional parts until we've had some more extensive tests done with perhaps login specific pages to make the transition a little more seemless.